Documentation/kubernetes/compatibility.rst

```diff
@@ -15,7 +15,9 @@ All Kubernetes versions listed are compatible with Cilium:
 ============= =========================== ============================
  1.7           * `extensions/v1beta1`_    ``cilium.io/v2`` has a
  1.8           * `networking.k8s.io/v1`_  `CustomResourceDefinition`
- 1.9
```

Comment:
The problem with the table is that each line written in the rst will be a real line after rendering.

I would suggested to put it like:

```
+----------------------+---------------------------+----------------------------+
| k8s Version          | k8s NetworkPolicy API     | CiliumNetworkPolicy        |
+----------------------+---------------------------+----------------------------+
| 1.7                  | * `extensions/v1beta1`_   | ``cilium.io/v2`` has a     |
+----------------------+---------------------------+ `CustomResourceDefinition` |
| 1.8, 1.9, 1.10, 1.11 | * `networking.k8s.io/v1`_ |                            |
+----------------------+---------------------------+----------------------------+
```

Reply:
@aanm addressed your concern :)